scss/prototype/_display.scss

```diff
@@ -0,0 +1,50 @@
+// Foundation for Sites by ZURB
+// foundation.zurb.com
+// Licensed under MIT Open Source
+
+////
+/// @group prototype-display
+////
+
+/// Responsive breakpoints for display, defaulted to global-prototype-responsive-breakpoints.
+/// @type Boolean
+$prototype-display-responsive-breakpoints: $global-prototype-responsive-breakpoints !default;
+
+@mixin foundation-prototype-display {
+  @each $display in (none, inline, inline-block, block, table, table-cell) {
```

Comment:
@IamManchanda, I think that makes sense. This way it's more customizable. One thing to remember is to make sure the variables aren't too generic like `$display-map` as by the name alone, there's no indication that it's specifically for prototyping. Other than that :+1: